deepmd/pt/model/task/ener.py

```diff
@@ -63,118 +54,64 @@ def __init__(
         precision: str = DEFAULT_PRECISION,
         distinguish_types: bool = False,
         rcond: Optional[float] = None,
+        seed: Optional[int] = None,
         **kwargs,
     ):
         """Construct a fitting net for energy.
 
-        Args:
-        - ntypes: Element count.
-        - embedding_width: Embedding width per atom.
-        - neuron: Number of neurons in each hidden layers of the fitting net.
-        - bias_atom_e: Average enery per atom for each element.
-        - resnet_dt: Using time-step in the ResNet construction.
+        Parameters
```

Comment:
The constructor's parameters should be documented in the class docstring.
See https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring